hack/update-estimator-protobuf.sh

```diff
@@ -36,10 +49,11 @@ ${GOPATH}/bin/go-to-protobuf \
 # The `go-to-protobuf` tool will modify all import proto files in vendor, so we should use go mod vendor to prevent.
 go mod vendor
 
-SERVICE_PROTO_FILES=$(find . -name "service.proto")
+SERVICE_PROTO_FILES=$(find . -not -path *./_tmp/* -name "service.proto")
 
 for file in ${SERVICE_PROTO_FILES[*]}; do
   protoc \
+    --proto_path=${KARMADA_ROOT} \
```

Comment:
If deleting this line, it works.

Reply:
but on the CI machine, it didn't work, and the output error is 
```
2021/11/23 06:49:44 /home/runner/work/karmada/karmada/vendor/k8s.io/apimachinery/pkg/util/intstr/generated.proto: File does not reside within any path specified using --proto_path (or -I).  You must specify a --proto_path which encompasses this file.  Note that the proto_path must be an exact prefix of the .proto file names -- protoc is too dumb to figure out when two paths (e.g. absolute and relative) are equivalent (it's harder than you think).
2021/11/23 06:49:44 protoc -I . -I /home/runner/go/src -I ./vendor -I ./third_party/protobuf/ --gogo_out=/home/runner/go/src /home/runner/work/karmada/karmada/vendor/k8s.io/apimachinery/pkg/util/intstr/generated.proto
2021/11/23 06:49:44 Unable to generate protoc on k8s.io.apimachinery.pkg.util.intstr: exit status 1
```

Reply:
have no idea about why this error occurs. 😞

Reply:
on the ci machine, GOPATH is 
```
go env GOPATH
/home/runner/go
```

but according to the error logs
```
/home/runner/work/karmada/karmada/vendor/k8s.io/apimachinery/pkg/util/intstr/generated.proto: File does not reside within any path specified using --proto_path (or -I).  You must specify a --proto_path which encompasses this file.  Note that the proto_path must be an exact prefix of the .proto file names -- protoc is too dumb to figure out when two paths (e.g. absolute and relative) are equivalent (it's harder than you think).
2021/11/23 06:49:44 protoc -I . -I /home/runner/go/src -I ./vendor -I ./third_party/protobuf/ --gogo_out=/home/runner/go/src /home/runner/work/karmada/karmada/vendor/k8s.io/apimachinery/pkg/util/intstr/generated.proto
2021/11/23 06:49:44 Unable to generate protoc on k8s.io.apimachinery.pkg.util.intstr: exit status 1
```
the code is located in `/home/runner/work`, but not in the GOPATH directory. Is this related to this problem?
Any ideas about the failed reason? :) @Garrybest

Reply:
Let me take look.